doc/guides/diagnostic-tooling-support-tiers.md

```diff
@@ -0,0 +1,115 @@
+# Diagnostic Tooling Support Tiers
+
+Diagnostic tooling is important to the consumers of Node.js. It is used both
+in development and in production in order to investigate problems.  The failure
+of one of these tools may be as big a problem for an end user as a bug within
+the runtime itself.
+
+The Node.js project has assessed the tools and the APIs which support those
+tools. Each of the tools and APIs has been put them into one of
+the following tiers.  
+
+* Tier 1 - Mmust always be working for all
+  Current and LTS Node.js releases. A release will not be shipped if the test
+  suite for the tool/API is not green. To be considered for inclusion
+  in this tier it must have a good test suite and that test suite and a job
+  must exist in the Node.js CI so that it can be run as part of the release
+  process.  No commit to master should break this tool/API if the next
+  release is within 1 month.
```

Comment:
We need to defined what does it mean to "work". In some cases it's a true-false decision (llnode), but in others it is a matter of accuracy (Linux perf in 8.x for example). I assume you mean the latter, but I think it should be written down.

cc @yunong @mmarchini

Reply:
Since we'll require a test suite on Node.js CI for Tier 1 tools, maybe we could define "work" as "successfully running a CI for the tool's test suite"?

Reply:
agreed, updated to try to clarify that.